types/soap/index.d.ts

```diff
@@ -1,87 +1,188 @@
-// Type definitions for soap 0.18
+// Type definitions for soap 0.19
 // Project: https://www.npmjs.com/package/soap
-// Definitions by: Leo Liang <https://github.com/aleung>, Cage Fox <https://github.com/cagefox>
+// Definitions by:  Leo Liang <https://github.com/aleung>
+//                  Cage Fox <https://github.com/cagefox>
+//                  Onur Yildirim <https://github.com/onury>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import * as events from 'events';
+import { EventEmitter } from 'events';
 
-interface Security {
+export interface ISOAPMethod {
```

Comment:
I prefer to treat abbreviations as if they were lower case words. See [this](https://stackoverflow.com/a/27172000/94148) highest vote answer in SO.

`ISoapMethod` is more readable than `ISOAPMethod`.

This comment also apply to `WSDL`, e.g. `IWSDLBaseOptions`.